docs/reference-guide/modules/commands/pages/configuration.adoc

```diff
@@ -1,239 +1,544 @@
 = Configuration
 
-This page describes how you can configure your Aggregate model.
+This page describes how to configure command handling in your Axon application.
```

Comment:
This entire chapter is no longer just about command configuration, but touches upon all the detailed configuration options. I think these belong in a separate chapter to reference instead of living here. But, for this PR and for getting the Docs for AF5, let's leave it as is. I will make a tremendous amount of changes (as I did for the `command-handlers.adoc` too), as a lot of it is plain wrong.